docs/adr/0003-boolean-algebra.md

```
@@ -361,41 +367,33 @@ then pass { init_collection(ip: client.ip + "_" + ua_hash) }
-SecLang's `multiMatch` action changes how a collection-targeting condition evaluates —
-instead of stopping at the first match, it iterates all values and fires side-effects
-per match. This is currently modeled as a non-disruptive action, but it is semantically
-a condition quantifier.
+SecLang's `multiMatch` action changes how an operator evaluates a target that has transformations —
+instead of evaluating after applying the transformations, are checked against the operator before and after every transformation function. This is currently modeled as a non-disruptive action, but it is semantically
+a flow control mechanism.
 
-**Recommendation: `each()` as a condition-level quantifier (Option A).**
+**Recommendation: `multi_match` as a metadata property.**
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)